app/controllers/api/v4/application_controller.rb

```diff
@@ -76,11 +75,11 @@ def set_runner
   end
 
   def set_run
-    if params[:historic] == '1'
-      @run = Run.includes(:game, :category, :user, :histories, segments: [:histories]).find36(params[:run])
-    else
-      @run = Run.includes(:game, :category, :user, :segments).find36(params[:run])
-    end
+    @run = if params[:historic] == '1'
+             Run.includes(:game, :category, :user, :histories, segments: [:histories]).find36(params[:run])
+           else
+             @run = Run.includes(:game, :category, :user, :segments).find36(params[:run])
+           end
```

Comment:
Looks like there's an extra `@run = ` in here (although it incidentally probably still works as intended)